benchmarks/serialization/benchmarker.rb

```diff
@@ -0,0 +1,48 @@
+require 'puppet'
+
+class Benchmarker
+  def initialize(target, size)
+    @direction = ENV['SER_DIRECTION'] == 'generate' ? :generate : :parse
+    @format = ENV['SER_FORMAT'] == 'pson' ? :pson : :json
+
+    puts "Benchmarker #{@direction} #{@format}"
+  end
+
+  def setup
+  end
+
+  def generate
+    path = File.expand_path(File.join(__FILE__, '../catalog.json'))
+    puts "Using catalog #{path}"
+
+    @data = File.read(path)
+    @catalog = JSON.parse(@data)
+  end
+
+  def run(args=nil)
+    0.upto(ENV['ITERATIONS'].to_i) do |i|
```

Comment:
I think the benchmark run task [already iterates over `ENV['ITERATIONS']`](https://github.com/puppetlabs/puppet/blob/master/tasks/benchmark.rake#L43) - just confirming we want to separately iterate up to the count here, ie ultimately producing parse/generate results for ENV['ITERATIONS']^2 iterations?